demos/esc/README.md

```diff
@@ -0,0 +1,85 @@
+# ESC Library Demo - eurus-project
+
+This is the example application demonstrating usage of the 
+Eurus Flight Controller (EFC) ESC (Electronic Speed Controller)
+library within the Zephyr framework.
+
+## Features & Demo configuration
+
+The library supports multiple ESC protocols, including:
+- Classic PWM
+- Oneshot 125
+- Oneshot 42
+- Multishot
+
+Default ESC protocol in this sample is PWM, but it can be reconfigured 
+using menuconfig. To start menuconfig, the demo needs to be built with: 
+
+`west build -b <board_name> <path_to_demo>`
+
+then, to start menuconfig enter:
+
+`west build -t menuconfig`
+
+![Eurus Flight Controller - ESC Demo menuconfig](.docs/menuconfig1.png)
+
+Next, by choosing ESC Protocol Selection, submenu will open where ESC PWM
```

Comment:
We don't need to teach people how to use menuconfig, this entire section can be removed. It's enough to list available protocols and variables and how to enter menuconfig.